_maps/shuttles/shiptest/syndicate_geneva.dmm

```diff
@@ -0,0 +1,2531 @@
+	dir = 4
+	},
+/turf/open/floor/plasteel/dark,
+/area/ship/crew/library)
+"dh" = (
+/obj/machinery/atmospherics/components/unary/vent_pump/on/layer4,
+/turf/open/floor/grass,
+/area/ship/crew/hydroponics)
+"dj" = (
+/obj/machinery/chem_master,
+/obj/structure/cable{
+	icon_state = "0-2"
+	},
+/obj/machinery/power/apc/auto_name/north,
+/turf/open/floor/grass,
+/area/ship/crew/hydroponics)
+"do" = (
+/obj/item/kirbyplants/random,
+/obj/item/vending_refill/medical,
+/obj/effect/turf_decal/corner/bottlegreen/border{
+	dir = 9
+	},
+/obj/machinery/atmospherics/components/unary/vent_pump/on/layer4,
+/obj/machinery/firealarm{
+	dir = 4;
+	pixel_x = -24
+	},
+/obj/item/vending_refill/autodrobe,
+/obj/effect/turf_decal/syndicateemblem/top/left,
+/turf/open/floor/mineral/plastitanium/red,
+/area/ship/hallway/central)
+"dr" = (
+/obj/item/kirbyplants/random,
+/obj/item/seeds/replicapod,
+/obj/effect/turf_decal/corner/bottlegreen/border{
+	dir = 5
+	},
+/obj/effect/turf_decal/syndicateemblem/top/right,
+/turf/open/floor/mineral/plastitanium/red,
+/area/ship/hallway/central)
+"dw" = (
+/obj/item/kirbyplants/random,
+/obj/effect/turf_decal/corner/bottlegreen{
+	dir = 9
+	},
+/obj/machinery/airalarm/directional/west,
+/obj/effect/decal/cleanable/dirt/dust,
+/turf/open/floor/plasteel/dark,
+/area/ship/crew/library)
+"dy" = (
+/obj/machinery/washing_machine,
+/obj/structure/cable{
+	icon_state = "0-2"
+	},
+/obj/machinery/power/apc/auto_name/north,
+/obj/machinery/firealarm{
+	dir = 4;
+	pixel_x = -24
+	},
+/turf/open/floor/carpet/nanoweave/red,
+/area/ship/crew/dorm)
+"dz" = (
+/obj/effect/turf_decal/corner/bottlegreen/three_quarters{
+	dir = 1
+	},
+/obj/machinery/light{
+	dir = 8
+	},
+/obj/machinery/door/firedoor/border_only,
+/obj/effect/decal/cleanable/dirt/dust,
+/turf/open/floor/plasteel/white,
+/area/ship/medical)
+"dP" = (
+/obj/structure/closet/cabinet,
+/obj/item/clothing/under/syndicate/skirt,
+/obj/item/clothing/under/syndicate/skirt,
+/obj/item/clothing/under/syndicate/skirt,
+/obj/item/clothing/under/syndicate/skirt,
+/obj/item/clothing/under/syndicate/skirt,
+/obj/item/clothing/under/syndicate,
+/obj/item/clothing/under/syndicate,
+/obj/item/clothing/under/syndicate,
+/obj/item/clothing/under/syndicate,
+/obj/item/clothing/under/syndicate,
+/obj/item/clothing/under/syndicate/bloodred/sleepytime,
+/obj/item/clothing/under/syndicate/bloodred/sleepytime,
+/obj/item/clothing/under/syndicate/bloodred/sleepytime,
+/turf/open/floor/carpet/nanoweave/red,
+/area/ship/crew/dorm)
+"dW" = (
+/obj/machinery/vending/hydroseeds{
+	all_items_free = 1
+	},
+/obj/machinery/light{
+	dir = 1
+	},
+/turf/open/floor/grass,
+/area/ship/crew/hydroponics)
+"dZ" = (
+/obj/effect/decal/cleanable/dirt,
+/turf/open/floor/grass,
+/area/ship/crew/hydroponics)
+"eh" = (
+/obj/effect/spawner/structure/window/reinforced,
+/obj/machinery/door/poddoor/preopen{
+	id = "spitfireshutters"
+	},
+/turf/open/floor/plating,
+/area/ship/crew/dorm)
+"ej" = (
+/obj/structure/sink/puddle,
```

Comment:
Out of all things (sink, high-capacity water tank, what-have-you) you put a puddle on a spaceship?

Reply:
> Out of all things (sink, high-capacity water tank, what-have-you) you put a puddle on a spaceship?

Yes.

Reply:
this is not changing

Reply:
Why

Reply:
After all, why not? Why shouldn't it be a puddle?

Reply:
the solgovvie wants their little splishy pool

Reply:
Doesn't really fits the theme of a spaceship but I guess this reasoning can't be beaten

> solgovvie

but it's a syndicate ship

Reply:
nsv is from solgov and is natural science, Lore Not Read